base/abstractset.jl

```diff
@@ -339,6 +339,8 @@ false
 ⊊, ⊋
 
 ⊊(a::AbstractSet, b::AbstractSet) = length(a) < length(b) && a ⊆ b
+⊊(a::AbstractSet, b) = a ⊊ Set(b)
+⊊(a, b::AbstractSet) = Set(a) ⊊ b
```

Comment:
Or maybe this would be better?

```suggestion
⊊(a::T, b) where {T <: AbstractSet} = a ⊊ T(b)
⊊(a, b::T) where {T <: AbstractSet} = T(a) ⊊ b
```

Reply:
that would assume the Sets can be losslessly converted to the other, but it would be valid (if unlikely) to compare a Vector{String} and a Set{Int}, for example

Reply:
Good point! I was thinking about `a` being a BitSet, but didn't think about failing conversions.